[SRVOCF-442] Move on-cluster function building to separate assembly; add the section on specifying which function to build (#51183)
openshift/openshift-docs: serverless/functions/serverless-functions-on-cluster-builds.adoc

:_content-type: ASSEMBLY
[id="serverless-functions-on-cluster-builds"]
= On-cluster function building and deploying
:context: serverless-functions-on-cluster-builds
include::_attributes/common-attributes.adoc[]

toc::[]

> Instead of building the function locally, you can build the function directly on the cluster. In this workflow, on your development machine, you only need to work with the function source code. This is useful, for example, when you cannot have the necessary function building tools installed, such as docker or podman.

Reviewer: ```suggestion
Instead of building a function locally, you can build a function directly on the cluster. When using this workflow on a local development machine, you only need to work with the function source code. This is useful, for example, when you cannot install on-cluster function building tools, such as docker or podman.
```

Reviewer: Check how we refer to Docker here also, idk if it needs a capital letter etc in OCP docs
